applications/teletype/src/teletype_bindings.erl

```diff
@@ -53,7 +53,14 @@ start_modules([]) ->
 notification(JObj) ->
     kz_util:put_callid(JObj),
     {EventCategory, EventName} = kz_util:get_event_type(JObj),
+    ShouldHandle = teletype_util:should_handle_notification(JObj),
```

Comment:
What happens if we make this an assertion instead?

```erlang
notification(JObj) ->
    'true' = teletype_util:should_handle_notification(JObj),
    ...
```

So the spawned process will just crash immediately if it won't handle the payload.

Reply:
I don't agree, it produce not necessary CRASH REPORT log for a case which teletype is not the default notify app.

Reply:
We have assertions in other event handlers for validating the payloads `true = kapi_whatever:payload_v(JObj),` so it isn't unprecedented. But I don't have a burning desire to see this change; just suggesting it for happy-case programming vs defensive programming.